Library/Formula/ht.rb

```diff
@@ -2,13 +2,13 @@
 
 class Ht < Formula
   homepage 'http://hte.sf.net/'
-  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.0.22.tar.bz2'
-  sha256 'c729d64bf7de440c7b1021d3d6657ccbdb103541b4082a58dca7c8402c773f58'
+  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.1.0.tar.bz2'
+  sha256 '31f5e8e2ca7f85d40bb18ef518bf1a105a6f602918a0755bc649f3f407b75d70'
 
   depends_on 'lzo'
 
   def install
-    system "chmod +x ./install-sh"
+    FileUtils.chmod "+x", "./install-sh"
```

Comment:
Sorry, this can have the `FileUtils.` omitted because it's included into the `Formula` class.